documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,293 @@
+
+.Get firmware detection by virt-v2v
+
+A `virt-v2v` that converts a guest also produces an OFV with configuration based on the virtual disk, such as driver or firmware. `forklift-controller` processes the produced OFV's status of the conversion pod. link:https://issues.redhat.com/browse/MTV-759[(MTV-759)]
+
+.Delete PV when the provider is deleted
+
+Each OVA provider has an attached PV and PVC. The PVC deletes automatically when the provider is deleted. {project-short} now also deletes the PV when the provider is deleted. link:https://issues.redhat.com/browse/MTV-848[(MTV-848)]
+
+.End-to-end test for import OVA
+
+The {project-short} process performs an end to end test of the imported OVA. link:https://issues.redhat.com/browse/MTV-624[(MTV-624)]
+
+.Create OVA provider under all projects
+
+The OVA provider can be created under all projects, not only under the forklift or MTV project. link:https://issues.redhat.com/browse/MTV-681[(MTV-681,] link:https://issues.redhat.com/browse/MTV-715[MTV-715)]
+
+.Migrate from an OVA provider to a restricted project
+
+Migration from OVA provider to a restricted project succeeds past the `kubevirt` step. link:https://issues.redhat.com/browse/MTV-689[(MTV-689)]
+
+.OVA inventory watcher detects deleted files
+
+The OVA inventory watcher detects changes in files, including those that have been deleted. Updates from the `ova-provider-server` pod are sent every five minutes to the `forklift-controller` pod that updates the inventory. link:https://issues.redhat.com/browse/MTV-733[(MTV-733)]
+
+.Migration from OVA validation rules
+
+VM migrations from OVA are validated according to vSphere. The migration can use a similar structure and inventory variable names as vSphere, of which the OVA model is a subset. The rules do not apply to clusters, hosts, the running state of the VMs, or any configuration that is not relevant for the OVA VMs. link:https://issues.redhat.com/browse/MTV-669[(MTV-669)]
+
+.VM cannot find the bootable disk when source VMware provider uses VDDK 8
+
+When migrating from a source provider that uses VDDK (VMware Virtual Disk Development Kit) version 8, the migration completes successfully, but the converted guest cannot boot. The guest cannot find the boot disk. 
+
+For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430274[Resolved Issues] in Jira.
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Deleting a migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs, and data volumes. You must archive a migration plan before deleting it, so you can clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
```

Comment:
Are there any other temporary resources than those mentioned here? If not, I'd recommend making a bulleted list out of them for better readability :-) WDYT?

```suggestion
Deleting a migration plan does not remove the following temporary resources:

* Importer pods
* Conversion pods
* Config maps
* Secrets
* Failed virtual machines (VMs)
* Data volumes

To be able to clean up temporary resources, archive the migration plan before deleting it. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
```